test/extended/testdata/builds/cluster-config/registry-blacklist.yaml

```diff
@@ -6,4 +6,3 @@ spec:
   registrySources:
       blockedRegistries:
       - docker.io
-      - quay.io
```

Comment:
per @runcom etcd had an init container which was not managed by the payload, pulling from quay.io. Blacklisting quay led to the MCO to roll out new nodes blocking quay.io, and subsequently caused the cluster to commit suicide. BZ filed to fix this.

Reply:
@adambkaplan did you file the BZ? I haven't cause there's some conversation already going on (I'll tag you on slack)

Reply:
>  Blacklisting quay led to the MCO to roll out new nodes blocking quay.io, and subsequently caused the cluster to commit suicide. BZ filed to fix this.

what will the BZ actually fix/change about the behavior?  disconnected installs might not need to reference quay.io and might have reason to blacklist it.

seems like all we can really do is:
1) not set it as blocked it in our tests
2) make sure the documentation around white/blacklists calls this out

Reply:
(if the BZ was to fix etcd get the image from the payload then that makes sense and you can ignore me)

Reply:
the only BZ I'm aware of was the quay blocking issue causing cluster to collapse and I closed that as working as expected. Fixing the etcd image to be in the release payload is another issue that people are already working on (I've been told)

Reply:
I was referring to the BZ related to the etcd image not being in the payload.
Filed https://bugzilla.redhat.com/show_bug.cgi?id=1686509 for this discovery.

Reply:
BZ for the etcd image not being in the payload: https://bugzilla.redhat.com/show_bug.cgi?id=1684670